### PR TITLE
reformatted/fixed grammar in footer

### PR DESCRIPTION
**Pull request by rnucuta**

### CODE REVIEW

File: resources/_gen/assets/css/css/style.css_d3f53f09220d597dac26fe7840c31fc9.content [hunk 1]
```diff
@@ -1 +1 @@
...
...}.resp-sharing-button--facebook:hover,.resp-sharing-button--facebook:active{background-color:#2d4373;border-color:#2d4373}.resp-sharing-button--twitter{background-color:#55acee;border-color:#55acee}.resp-sharing-button--twitter:hover,.resp-sharing-button--twitter:active{background-color:#2795e9;border-color:#2795e9}.resp-sharing-button--google{background-color:#dd4b39;border-color:#dd4b39}.resp-sharing-button--google:hover,.resp-sharing-button--google:active{background-color:#c23321;border-color:#c23321}.resp-sharing-button--email{background-color:#777;border-color:#777}.resp-sharing-button--email:hover,.resp-sharing-button--email:active{background-color:#5e5e5e;border-color:#5e5e5e}.resp-sharing-button--reddit{background-color:#5f99cf;border-color:#5f99cf}.resp-sharing-button--reddit:hover,.resp-sharing-button--reddit:active{background-color:#3a80c1;border-color:#3a80c1}.resp-sharing-button--whatsapp{background-color:#25d366;border-color:#25d366}.resp-sharing-button--whatsapp:hover,.resp-sharing-button--whatsapp:active{background-color:#1da851;border-color:#1da851}.resp-sharing-button--telegram{background-color:#54a9eb}.resp-sharing-button--telegram:hover{background-color:#4b97d1}
```
Comment (CLiu13) [inline resources/_gen/assets/css/css/style.css_d3f53f09220d597dac26fe7840c31fc9.content]: Why was this file modified?

Comment (rnucuta) [reply resources/_gen/assets/css/css/style.css_d3f53f09220d597dac26fe7840c31fc9.content]: Changed "Design And Developed by Ecotivity . Copyright © 2019. All Rights Reserved." to "Designed And Developed by Ecotivity. Copyright © 2019. All Rights Reserved." This fixed a spacing error and a grammar mistake.
